pkg/api/server/v1alpha2/results_test.go

```diff
@@ -102,6 +104,69 @@ func TestCreateResult(t *testing.T) {
 	}
 }
 
+func TestUpdateResult(t *testing.T) {
+	srv, err := New(test.NewDB(t))
+	if err != nil {
+		t.Fatalf("failed to create server: %v", err)
+	}
+	ctx := context.Background()
+
+	tt := []struct {
+		name    string
+		create  *pb.Result
+		update  *pb.Result
+		expect  *pb.Result
```

Comment:
We should add a comment here explaining the behavior of this field - this isn't the Result we expect, but rather the diff from the initially created Result.